src/rgw/rgw_perf_counters.cc

```diff
@@ -61,8 +61,8 @@ int rgw_perf_start(CephContext *cct)
   plb.add_u64_counter(l_rgw_pubsub_missing_conf, "pubsub_missing_conf", "Pubsub events could not be handled because of missing configuration");
   
   plb.add_u64_counter(l_rgw_lua_script_ok, "lua_script_ok", "Successfull executions of lua scripts");
-  plb.add_u64_counter(l_rgw_lua_script_fail, "lua_script_fail", "Failed executions of lua scripts");
-  plb.add_u64(l_rgw_lua_current_vms, "lua_current_vms", "Number of Lua VMs currently being executed");
+  plb.add_u64_counter(l_rgw_lua_script_fail, "lua_script_fail", "Failed executions of lua scripts", NULL, PerfCountersBuilder::PRIO_UNINTERESTING);
+  plb.add_u64(l_rgw_lua_current_vms, "lua_current_vms", "Number of Lua VMs currently being executed", NULL, PerfCountersBuilder::PRIO_UNINTERESTING);
```

Comment:
these look like useful metrics for a user of lua scripts

cc @yuvalif

Reply:
this is useful if the feature is used.
failures in lua scripts are not causing request failures, therefore the only way to know they failed is to use logs/counters

the "current vms" is to make sure that there is no resource leak, or overloading the system.
less important metric

Reply:
> What commands do you use to get this perfcounters

the admin socket 'perf dump' command

> how does the user get this perfcounters?

i believe the ceph dashboard integrates with grafana to display the data in prometheus. i don't know which counters are visible by default there, but i've assumed that it can be made to display any of them

Reply:
> the admin socket 'perf dump' command

That command doesn't filter priorities right? I just learned there exists the command `ceph daemonperf` which takes into account priorities but I find it pretty verbose and hard to understand. I think it would make more sense to have layouts of predefined percounters rather than useful priorities that could increase over time and make it harder to read:
example:
![image](https://user-images.githubusercontent.com/30913090/179071418-ea77a16d-fbce-4664-8130-3896643fb2b2.png)

> i believe the ceph dashboard integrates with grafana to display the data in prometheus. i don't know which counters 
> are visible by default there, but i've assumed that it can be made to display any of them

I proposed this on another chat but I think that it could be great to have a page in the docs where you could see all perfcounters and grafana metrics available so that you could check whether a perfcounter is avaiable. If not available, the page could have a link to request a change in priority in the specific component.